contrib/python-beautifulsoup4/template.py

```diff
@@ -0,0 +1,22 @@
+pkgname = "python-beautifulsoup4"
+pkgver = "4.12.3"
+pkgrel = 0
+build_style = "python_pep517"
+hostmakedepends = [
+    "python-build",
+    "python-hatchling",
+    "python-installer",
+    "python-setuptools",
+    "python-wheel",
+]
+checkdepends = ["python-pytest"]
```

Comment:
this is missing a python-soupsieve runtime dep

Reply:
creates a cycilc dependency

Reply:
ok

Reply:
soupsieve tests depend on bs4, so it'd still create a cyclic dep

Reply:
i think this all can be easily avoided by making manually pull both together or maybe do an install_if in soupsieve for bs4